discovery-package/src/main/resources/zosmf-static-definition.yaml.template

```diff
@@ -40,7 +40,7 @@ services:
         - apiId: ibm.zosmf
           gatewayUrl: api/v1
           documentationUrl: https://www.ibm.com/support/knowledgecenter/en/SSLTBW_2.4.0/com.ibm.zos.v2r4.izua700/IZUHPINFO_RESTServices.htm
-          swaggerUrl: ${ZOSMF_SCHEME:-https}://${ZOSMF_HOST}:${ZOSMF_PORT}/zosmf/api/docs
+          swaggerUrl: ${ZWE_zOSMF_scheme:-https}://${ZOSMF_HOST}:${ZOSMF_PORT}/zosmf/api/docs
```

Comment:
is this related to cors?

Reply:
both `ZWE_zOSMF_host` and `ZOSMF_HOST` exist (same with ZOSMF_PORT), these are there because they are mandatory, scheme will be optional in the zOSMF section so `ZOSMF_SCHEME` will not exist.

Maybe for v3 we can move these to the new variable format besides deleting the unused zosmf definition.